.github/workflows/commit-format-check.yaml

```diff
@@ -0,0 +1,70 @@
+name: Commits Formatting Check
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [staging]
+
+jobs:
+  commit:
+    name: Commit and PR Format Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Commit Lines Length Check
+        if: ${{ github.actor != 'dependabot' && github.actor != 'dependabot[bot]' }}
+        uses: gsactions/commit-message-checker@v2
+        with:
+          pattern: '((^(?=(?:.|\n)*(?:^|\n)\[\d\]: .{69,}(?:$|\n)(?:.|\n)*)(?:.|\n)*$)|(^(?!(?:.|\n)*(?:^|\n).{74,}(?:$|\n)(?:.|\n)*)(?:.|\n)*$))'
+          flags: ''
```

Comment:
'Match everything when lines are longer than 74 characters and begin with a citation or just shorter than 74 characters; otherwise, don't match anything'.
 
 We can change to different sizes if wanted/needed.
 
 Currently has an edge-case where it will allow commits with lines longer than 74-characters if there are citations also with 74+ characters